Source/WebCore/bindings/js/JSTrustedTypePolicyCustom.cpp

```diff
@@ -33,9 +33,9 @@ namespace WebCore {
 template<typename Visitor>
 void JSTrustedTypePolicy::visitAdditionalChildren(Visitor& visitor)
 {
-    RefPtr<CreateHTMLCallback> protectedCreateHTML;
-    RefPtr<CreateScriptCallback> protectedCreateScript;
-    RefPtr<CreateScriptURLCallback> protectedCreateScriptURL;
+    CreateHTMLCallback* protectedCreateHTML = nullptr;
```

Comment:
1. Don't we need to silence static analysis for now
2. Calling this "protected*" seems wrong now that it is no longer protected

Reply:
oh good point.